README.md

```diff
@@ -1,4 +1,4 @@
-# Astrobee Robot Software
+# Astrobee Robot Software v1
```

Comment:
---
**Sema Reaction:** :trophy: This code is awesome

Reply:
test comment

Reply:
Fix not maintainable. Change the variable

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not maintainable

Reply:
Looks good maintainable.

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Maintainable

Reply:
Test

---
**Sema Reaction:** :ok_hand: This code looks good | **Sema Tags:** Readable, Not reusable, Maintainable

Reply:
Functional requirements not met - Business Rules
Do not use displayName for naming components. Instead, name the component by reference.

📄 [AirBnB](https://github.com/airbnb/javascript/tree/master/react)

Reply:
elegant

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Readable, Secure, Inefficient

Reply:
looks great!

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

Reply:
awesome effective 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient

Reply:
1 comment to see link

Reply:
It works?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Secure, Efficient

Reply:
test one 8/1